source/pdo_sqlsrv/config.m4

```diff
@@ -63,7 +63,10 @@ if test "$PHP_PDO_SQLSRV" != "no"; then
            shared/StringFunctions.cpp \
            "
   AC_MSG_CHECKING([for PDO_SQLSRV headers])
-  if test -f $srcdir/ext/pdo_sqlsrv/shared/core_sqlsrv.h; then
+  if test -f $srcdir/ext/sqlsrv/shared/core_sqlsrv.h; then
+    pdo_sqlsrv_inc_path=$srcdir/ext/sqlsrv/shared/
+    shared_src_class=""
+  elif test -f $srcdir/ext/pdo_sqlsrv/shared/core_sqlsrv.h; then
```

Comment:
> This change makes it impossible to compile just pdo_sqlsrv without sqlsrv (I get a bunch of undefined reference compilation errors)

Can you elaborate a bit what you tried to do? This should only happen if you copy both extensions to the PHP source `ext/` directory and then specify `--disable-sqlsrv`. As this should be required rarely I thought it might be better than no support. But it is probably possible to test if `sqlsrv` is enabled, [like this](https://github.com/Microsoft/msphpsql/blob/master/source/pdo_sqlsrv/config.m4#L25). I'll have a look.

Reply:
@david-puglielli I've added the additional check. Had to swap between the extensions, because `pdo_sqlsrv` is included first by `buildconf`. This might be a bit fragile, but enables me to build both without arguments and with `--with-pdo_sqlsrv --disable-sqlsrv` and also with both extensions active. Does this address your concerns?

Reply:
Ok, I think your recent changes address my concerns. I've tested every combination, and the only one that doesn't work is pdo_sqlsrv shared and sqlsrv static, but I don't think that's a big problem. We will continue testing and see if we can get it working on Windows as well.